edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/edgedeployment/deployment/KubernetesDeploymentMapper.cs

```diff
@@ -189,6 +201,8 @@ V1PodTemplateSpec GetPod(string name, IModuleIdentity identity, KubernetesModule
             Option<List<V1ContainerPort>> exposedPorts = module.Config.CreateOptions.ExposedPorts
                 .Map(PortExtensions.GetContainerPorts);
 
+            Option<V1ResourceRequirements> resourceRequirements = this.PrepareModuleResourceRequirements(module, identity);
```

Comment:
What if one update the environment variable after container is started, are we able to pick up/refresh the limit?